src/main/java/seedu/address/logic/commands/SearchPatientCommand.java

```diff
@@ -0,0 +1,197 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TEMP_RANGE;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.TemperatureRange;
+
+
+/**
+ * Search a person or a list of person according to a name or a range of temperature.
+ */
+public class SearchPatientCommand extends Command {
+
+    public static final String COMMAND_WORD = "searchpatient";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Search a patient in Covigent. "
+            + "Parameters: "
+            + PREFIX_NAME + "NAME || "
+            + PREFIX_TEMP_RANGE + "TEMPERATURERANGE "
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_NAME + "John Doe ";
```

Comment:
Personally, I think including an example for search via temperature range may be better since this may be more difficult for users to understand how to use.

Reply:
sure